### DISCUSSION TITLE
How to use kittens on Windows

### DISCUSSION THREAD

**Discussion by jeansagi** [kovidgoyal/kitty#6717] [Q&A]
Hi,

I'm new to **kitty** and installed it with chocolatey on windows.
So far everything worked as expected.

Reading the documentation, I read about kittens and was particularly interested in **icat**.
But it seems not to work on windows.

I tried:

- kitten icat image.png
- kitty +kitten icat image.png

And none of above worked.

So,

Does kittens work on Windows 11?
Is there a procedure to make icat works with kitty.

Really appreciate any help.

J.

**Comment by Luflosi** [comment]
This project is not available via chocolatey because it doesn't natively run on windows.
What you installed is a different and unrelated project with an almost identical name: https://github.com/cyd01/KiTTY.

**Comment by jeansagi** [comment]
Ouch...
My bad. Really sorry to bother.

BTW: Really *COOL* terminal !

Perhaps someday you will consider porting it to Windows (If so, count me on
for testing ;) ).


J.


El dom, 15 oct 2023 a las 16:57, Luflosi ***@***.***>)
escribió:

> This project is not available via chocolatey because it doesn't natively
> run on windows.
> What you installed is a different and unrelated project with an almost
> identical name: https://github.com/cyd01/KiTTY.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kovidgoyal/kitty/discussions/6717#discussioncomment-7287060>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFY3AELXGNDQMSRLJ3GXLRLX7RL35AVCNFSM6AAAAAA6BGNC3KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TEOBXGA3DA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

J.
